tests/zeekjs/hook/no-break.sh

```diff
@@ -37,4 +37,10 @@ redef LogAscii::use_json = T;
 @ifdef ( Site::private_address_space_is_local )
 redef Site::private_address_space_is_local = F;
 @endif
+
+# Don't log ip_proto - use IP::protocol_names as indicator
+# if Conn::Info$ip_proto exists.
+@ifdef ( IP::protocol_names )
+redef record Conn::Info$ip_proto -= { &log };
+@endif
```

Comment:
Nit: I think I personally would prefer loading the new policy script to disable the logging since it would be more self-documenting and less finicky. Like discussed offline this still needs a preproc check whether the script can be loaded.

Here and below.

Reply:
Will update. thanks!